Mlem/App/Views/Root/Tabs/Settings/AccessibilitySettingsView.swift

```diff
@@ -32,12 +33,28 @@ struct AccessibilitySettingsView: View {
                         systemImage: Icons.read,
                         destination: .settings(.postReadIndicator)
                     )
+                } header: {
+                    Text("Differentiate Without Color")
                 }
             }
             
             Section {
                 Toggle("Website Thumbnail Indicator", systemImage: Icons.browser, isOn: $websiteThumbnailIcon)
                 Toggle("Settings Icons", systemImage: Icons.icon, isOn: $showSettingsIcons)
+            } header: {
+                Text("Non-Text Indicators")
+            }
+            
+            Section {
+                NavigationLink(
+                    "Drag-To-Zoom Images",
```

Comment:
IMO we shouldn't hyphenate this. Apple doesn't include hyphens in similar circumstances:
<img src="https://github.com/user-attachments/assets/00103ca8-65dd-420d-be63-15ac07118d85" width=50%>
```suggestion
                    "Drag to Zoom Images",
```

I also wonder whether "Swipe to Zoom" might be better? "Drag" suggests that you're moving something, which isn't the case here. Though "Swipe" sounds like one smooth motion after which you let go, which isn't the case here either... I'm not sure.